src/permalink.js

```diff
@@ -29,6 +29,10 @@ var Permalink = function () {
     //Check for content permalink
     var content = uriInterpreter.getContentPermalink();
     if (content) {
+        if(content.contentId && content.contentId.indexOf('lb-post') >= 0)  {
```

Comment:
Is it possible that `lb-post` could be at the beginning of the string? e.g. `>= -1`